cheddar/cheddar-integration-aws/src/test/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/DynamoDocumentStoreTemplateTest.java

```diff
@@ -121,6 +122,38 @@ public void shouldCreate_withItem() {
     }
 
     @Test
+    public void shouldCreate_withItemWithNullProperty() {
+        // Given
+        final ItemId itemId = new ItemId(randomId());
+        final StubItem stubItem = generateRandomStubItem(itemId);
+        stubItem.setStringProperty2(null);
+
+        final ItemConfiguration itemConfiguration = new ItemConfiguration(StubItem.class, tableName);
+        final Collection<ItemConfiguration> itemConfigurations = Arrays.asList(itemConfiguration);
+        when(mockDatabaseSchemaHolder.itemConfigurations()).thenReturn(itemConfigurations);
+
+        final Table mockTable = mock(Table.class);
+        when(mockDynamoDBClient.getTable(any(String.class))).thenReturn(mockTable);
+
+        final DynamoDocumentStoreTemplate dynamoDocumentStoreTemplate = new DynamoDocumentStoreTemplate(
+                mockDatabaseSchemaHolder);
+        dynamoDocumentStoreTemplate.initialize(mockAmazonDynamoDbClient);
+
+        final Item mockTableItem = mock(Item.class);
+        when(mockTableItem.toJSON()).thenReturn(dynamoDocumentStoreTemplate.itemToString(stubItem));
+
+        // When
+        dynamoDocumentStoreTemplate.create(stubItem);
+
+        // Then
+        final ArgumentCaptor<PutItemSpec> getItemRequestCaptor = ArgumentCaptor.forClass(PutItemSpec.class);
+        verify(mockTable).putItem(getItemRequestCaptor.capture());
+
+        final PutItemSpec spec = getItemRequestCaptor.getValue();
+        assertFalse(spec.getItem().isNull("stringProperty2"));
```

Comment:
Ok, fair enough.  The `isNull` method does the following `Returns true if the specified attribute exists with a null value; false otherwise.` which does check the problem we have here but I can understand something like the `hasAttribute` method helps convey what is being tested in a better way.